### PR TITLE
added product listing

### PR DESCRIPTION
**Pull request by sanjay-xdr**
added : products
demo:- https://objective-jackson-0fc355.netlify.app/pages/product/product

### CODE REVIEW

File: Pages/Product/product.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  width: 100%;
+  background-color: var(--white-color);
+}
+
+.icon-heart,
+.icon-cart {
+  font-size: 3rem;
+}
+
+.filters{
```
Comment (tushargoyal812) [inline Pages/Product/product.css:24]: suggestion: add space before {

**Review by tushargoyal812 — COMMENTED** [review]
just minor changes and good to merge

File: Pages/Product/product.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    flex: 20;
```
Comment (mohitdhatrak) [inline Pages/Product/product.css:25]: what does this property do i see no change on removing it?

File: Pages/Product/product.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    flex: 80;
+}
+
+input[type=checkbox],input[type=radio]{
+    height: 2rem;
+    width: 4rem;
+}
+.btn-shopping-cart:hover{
+    color: black;
+}
```
Comment (mohitdhatrak) [inline Pages/Product/product.css]: use hexcode here too, #000

File: Pages/Product/product.html [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+          <span class="icon-text">Cart</span>
+        </div>
+      </div>
+    </nav>
+
+    <main class="container-flex">
+      <section class="filters">
+        <div class="container-flex flex-sa">
+          <h2 class="font-l">Filters</h2>
+          <p class="font-medium">Clear</p>
```
Comment (mohitdhatrak) [inline Pages/Product/product.html:44]: use semantic tag like button for clear, not p

File: Pages/Product/product.css (hunk 2 above)
Comment (sanjay-xdr) [reply Pages/Product/product.css:25]: its for flex grow dividing the page into two part

**Review by mohitdhatrak — COMMENTED** [review]
Good work, saw that you are using section tag instead of div everywhere.

**Pull request merged**